src/EnergyPlus/HeatingCoils.cc

```diff
@@ -312,7 +312,7 @@ namespace HeatingCoils {
         UpdateHeatingCoil(state, CoilNum);
 
         // Report the current HeatingCoil
-        ReportHeatingCoil(state, CoilNum);
+        ReportHeatingCoil(state, CoilNum, CoilIsSuppHeater);
```

Comment:
Maybe this new argument is what allows the calculation to be deferred for supplemental heater operation.

Reply:
Yes, this allows the coil model to save the power number for heating coil and supp coil so when these are added in the parent the correct value is used. Before, the electric supp heater would have overwritten DataHVACGlobals::ElecHeatingCoilPower which is why I needed the new global DataHVACGlobals::SuppHeatingCoilPower.